code/modules/mob/living/carbon/human/human_organs.dm

```diff
@@ -59,13 +59,19 @@
 	if(stance_damage < 0)
 		stance_damage = 0
 
+	if(stance_damage <= 6)
+		legless = FALSE
+
```

Comment:
Update_canmove() underneath legless here so you can walk again when legless is false